backend/app/api/routes/llm_speech_examples.md

```
@@ -0,0 +1,470 @@
+# Speech-to-Speech (STS) API Examples
+
+## Endpoint
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)